routing/unified_edges.go

```diff
@@ -222,6 +223,14 @@ func (u *edgeUnifier) getEdgeNetwork(amt lnwire.MilliSatoshi) *unifiedEdge {
 			continue
 		}
 
+		// Track the maximal capacity for usable channels. If we don't
+		// know the capacity, we fall back to MaxHTLC.
+		capMsat := lnwire.NewMSatFromSatoshis(edge.capacity)
+		if capMsat == 0 && edge.policy.MessageFlags.HasMaxHtlc() {
+			capMsat = edge.policy.MaxHTLC
```

Comment:
I think we would only like to do this if we are in `assumeChannelValid=false` (normal) mode, otherwise this could become very spammy - every channel would fall back to `MaxHTLC`. I could add a `Trace` statement here. Alternatively we have to pass in `assumeChannelValid`, which would perhaps be too much just for logging?

Reply:
yeah I think this only happens in neutrino. Using a `Trace` sounds good.